content/en/cosign/overview.md

```diff
@@ -63,24 +63,44 @@ $ cosign verify-blob <file> --bundle cosign.bundle --certificate-identity=name@e
 
 To verify, Cosign queries the transparency log (Rekor) to compare the public key bound to the certificate, and checks the timestamp on the signature against the artifact’s entry in the transparency log. The signature is valid if its timestamp falls within the small window of time that the key pair and certificate issued by the certificate authority were valid.
 
-### Working with containers
+## Example: Working with containers
 
-Signing and verifying a container is similar to working with blobs. The Cosign command to sign a container image is:
+> NOTE: In this example, we will create a container using [ttl.sh](https://ttl.sh/).  It offers free, short-lived (i.e., hours), anonymous container image hosting if you just want to try out signing and verifying commands.  
+
+To use `ttl.sh` and [crane](https://github.com/google/go-containerregistry/tree/main/cmd/crane) to prepare the image that we want to sign. Run the following:
+
+```
+$ SRC_IMAGE=busybox
+$ SRC_DIGEST=$(crane digest busybox)
+$ IMAGE_URI=ttl.sh/$(uuidgen | head -c 8 | tr 'A-Z' 'a-z')
+$ crane cp $SRC_IMAGE@$SRC_DIGEST $IMAGE_URI:1h
+$ IMAGE_URI_DIGEST=$IMAGE_URI@$SRC_DIGEST
+```
+
+### Keyless signing of a container
+
+The following code signs the created container image.  The command to sign container images is `cosign sign <image URI>`.  Note that for containers, there is no bundle as there is with blobs, as the signature and certificate are attached directly to the container:
```

Comment:
I would be consistent with the variable

```suggestion
The following code signs the created container image.  The command to sign container images is `cosign sign $IMAGE_URI_DIGEST`.  Note that for containers, there is no bundle as there is with blobs, as the signature and certificate are attached directly to the container:
```

Reply:
That's not exactly the command to sign images.   It's actually cosign sign <image URI> not $IMAGE_URI_DIGEST.   That is just the command for this example.  I can change the wording so it says it's the command for the example.

Reply:
I reworded to indicate that the example uses $IMAGE_URI_DIGEST.